### PR TITLE
Add documentation for Azure PVC support

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

**Comment by enys** [comment]
@LiseMachetel FYI following our discussion.

File: chart-value-examples/azure/values.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  enabled: true
+  ## redis-ha data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.
+  ## * managed-csi-premium provisions premium SSD disks (recommended)
+  ## * managed-csi provisions standard SSD-backed disks
+  ## Do not use azurefile based storage classes, they are not compatible.
+  storageClass: "managed-csi-premium" # ok
```
Comment (davidrapin) [inline chart-value-examples/azure/values.yaml:20]:
```suggestion
  ## Linkurious Enterprise data Persistent Volume Storage Class
  ## If defined here, sets storageClassName: <storageClass> in the generated PersistentVolumeClaim
  ## If set to "-" here, sets storageClassName: "", which disables dynamic provisioning
  ## If undefined (the default) or set to null, then, no storageClassName spec is
  ##   set, thus choosing the default provisioner.
  ## Possible values:
  ## * "managed-csi-premium" provisions premium SSD disks (recommended)
  ## * "managed-csi" provisions standard SSD-backed disks
  ## Do not use azurefile-based storage classes, they are not compatible.
  storageClass: "managed-csi-premium"
```

File: chart-value-examples/azure/values.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+image:
+  repository: hub.docker.nexus3.linkurious.net/linkurious/linkurious-enterprise
```
Comment (fantandrea) [inline chart-value-examples/azure/values.yaml:2]: @enys are you sure you want to keep the URL here?

Comment (enys) [reply chart-value-examples/azure/values.yaml:2]: We do not. Good catch

Comment (enys) [reply chart-value-examples/azure/values.yaml]: This has been fixed, thx

**Review by davidrapin — APPROVED** [review]

**Pull request merged**